src/test/java/no/unit/nva/bare/BareConnectionTest.java

```diff
@@ -73,10 +78,16 @@ public void testExceptionOnBareConnection() throws IOException {
 
     @Test
     public void testConnect() throws IOException {
-        URL invalidUrl = Paths.get("/dev/null").toUri().toURL();
+        String test = "/dev/null";
+        File testFile = new File(test);
+        if (!testFile.exists()) {
+            Files.createDirectories(Paths.get(test));
+        }
+        URL invalidUrl = Paths.get(test).toUri().toURL();
         BareConnection bareConnection = new BareConnection();
         final InputStreamReader connect = bareConnection.connect(invalidUrl);
         assertNotNull(connect);
+        assertTrue(testFile.delete());
```

Comment:
Remove AssertTrue(...)  and just delete file - test fails on linux and it doesn't matter